apps/web/examples/newPage/example.controller.js

```diff
@@ -0,0 +1,101 @@
+import logger from '../../src/server/lib/logger.js';
+import * as exampleService from './example.service.js';
+
+/** @typedef {import('./example.types').Example} Example */
+
+/**
+ * @typedef {object} ViewAllExamplesRenderOptions
+ * @property {Example[]} allExamplesFormatted
+ */
+
+/** @type {import('@pins/express').RenderHandler<ViewAllExamplesRenderOptions>}  */
+export async function viewAllExamples(request, response) {
+	const allExamples = await exampleService.getAllExample();
+
+	// Best to check that we have data, as we likely will not be able to display a page if we don't
+	if (allExamples) {
+		// Question: should we create mapper to contain the code below
+		const allExamplesFormatted = [];
+
+		for (const example of allExamples) {
+			if (example) {
+				const formattedExample = {
+					id: example.id,
+					value: example.value
+				};
+
+				allExamplesFormatted.push({ ...formattedExample });
+			}
+		}
+
+		return response.render('./example.njk', { allExamplesFormatted });
+	}
+
+	return response.render('../../src/server/views/app/404.njk');
+}
+
+/**
+ *
+ * @typedef {object} ViewExampleRenderOptions
+ * @property {Example} exampleFormatted
+ */
+
+/** @type {import('@pins/express').RenderHandler<ViewExampleRenderOptions>}  */
+export async function viewExampleById(request, response) {
+	const example = await exampleService.getExampleById(request.params.id);
+
+	// Best to check that we have data, as we likely will not be able to display a page if we don't
+	if (example) {
+		// Question: should we create mapper to contain the code below
+		const exampleFormatted = {
+			id: example.id,
+			value: example.value
+		};
+
+		return response.render('./example.njk', { exampleFormatted });
+	}
+
+	return response.render('../../src/server/views/app/404.njk');
```

Comment:
We should probably add some shortcuts to common views, like a dictionary with 404, 500, etc. or even have some extension methods (are those possible in js?) such as `response.renderNotFound()`, `response.renderError()`...

Reply:
maybe thinking outside the scope of this PR...

Reply:
I agree it would be nice to have the views in a dictionary or similar - on front office there's a "view" file in "lib" which handles this (helps with reuse/decoupling)

Reply:
> We should probably add some shortcuts to common views, like a dictionary with 404, 500, etc. or even have some extension methods (are those possible in js?) such as `response.renderNotFound()`, `response.renderError()`...

Extension methods would need to use prototype inheritance, so are out...